spec/controllers/two_factor_authentication/piv_cac_verification_controller_spec.rb

```diff
@@ -11,28 +11,33 @@
   let(:nonce) { 'once' }
 
   let(:x509_subject) { 'o=US, ou=DoD, cn=John.Doe.1234' }
+  let(:x509_issuer) { 'foo' }
```

Comment:
Can we get a more realistic value here? I feel like a lot of our PIV code is super unclear because we don't have very realistic example values, so it's hard to document and know what to expect